### ISSUE TITLE
Outside vector image offset on paste

### ISSUE THREAD

**Issue by artisteacher** [turtletooth/tahoma#255]
**Describe the bug**
Pasting a vector image from outside of Tahoma pastes as a raster image with a good deal of offset - it might be severely cropped or not even visible.

**To Reproduce**
1. Create a basic vector shape in Adobe Illustrator - or equivalent vector program
2. Copy
3. In Tahoma, select a cell & paste

**Expected behavior**
I had expected that vector data would paste onto a vector level. More importantly, I expected the full image to be there without any offset/cropping.

**Screenshots**
Original image in Adobe Illustrator:
![Screen Shot 2020-09-29 at 8 19 45 AM](https://user-images.githubusercontent.com/21159116/94558066-7387de00-022d-11eb-8348-9ff6118a1a25.png)

Pasted image in Tahoma:
![Screen Shot 2020-09-29 at 8 19 35 AM](https://user-images.githubusercontent.com/21159116/94558095-7edb0980-022d-11eb-8d89-fb0124cc50f3.png)

**Device Information:**
 - OS: macOS 10.15.6
 - Tahoma Version: Alpha 1.1

**Comment by RodneyBaker** [comment]
> I had expected that vector data would paste onto a vector level.

Wouldn't that be a completely different feature than the current raster copy/paste feature?

**Comment by jeremybullock** [comment]
Vector import hasn't been implemented yet, but it is a good feature request.  We have some folks doing some great stuff in vector.  It would be cool to be able to import vectors as vectors.

**Comment by artisteacher** [comment]
Yeah, vector pasting as vectors would be super helpful. For some reason that's what I'd expected. Hopefully if this is implemented it could also resolve some of the SVG import issues.

**Comment by RodneyBaker** [comment]
An anticipated difficulty with importing 'vectors as vectors' short of creating a new level type is that part of the process would likely rely on the same conversion process that SVG to PLI conversion does.  That process is broke in several ways at present; primarily positioning (using top left corner vs center of all vectors) and scaling (where the imported vectors scale too large).
